wiki/osu!tourney/en.md

```diff
@@ -296,14 +292,13 @@ We may award badges to the first place winners of second (and further) iteration
 
 ## Troubleshooting
 
-### How do fresh install osu! without uninstalling my current?
+### How do I fresh install osu! without uninstalling my current?
```

Comment:
"How do I create a fresh osu! installation without uninstalling the current game?"